source/renderer/app/components/widgets/notification-dot/NotificationDot.scss

```diff
@@ -0,0 +1,12 @@
+.root {
+  position: relative;
+
+  .dot {
+    background-color: #ea4c5b;
```

Comment:
Should this use a theme variable instead?

Reply:
Yes, I'll update. Thanks!